.github/workflows/ubuntu.yml

```diff
@@ -114,22 +124,19 @@ jobs:
 
     - name: Add GoogleTest (Required by CppAD)
       run: |
-        sudo apt install libgtest-dev build-essential cmake
+        sudo apt install libgtest-dev build-essential cmake -y
```

Comment:
Can we figure out how to remove the dependency on GoogleTest. We shouldn't need to build the CppAD tests and GoogleTest should only be needed if we are building those.